### DISCUSSION TITLE
Is there a way to set default `staleTime` for `prefetchQuery()` using `new QueryClient({defaultOptions: {}})`?

### DISCUSSION THREAD

**Discussion by frederikhors** [TanStack/query#6036] [Q&A] [answered]
Is there a way to set default `staleTime` for `prefetchQuery()` using `new QueryClient({defaultOptions: {}})`?

**Comment by TkDodo** [comment]
I don't understand the question

**Reply by frederikhors** [reply]
Sorry. I explain it better.

I'm using Svelte and I'm trying to prefetch details pages on mouse hover on the list.

I'm always using the same value for `staleTime` in the below code. So I'm wondering if there isn't a way to set it by default.

```ts
queryClient.prefetchQuery({
  queryKey: playerKeys.byId(player_id),
  queryFn: playerById(player_id),
  staleTime: 5000,
});
```

**Reply by TkDodo** [reply, marked as answer]
`queryClient.setQueryDefaults(key, defaults)` will work
